.env

```
@@ -25,3 +25,9 @@ OJS_SYNC_ENABLED="false"
 # OJS_DATABASE_USER="root"
 # OJS_DATABASE_PASSWORD=""
 ALLOWED_ORIGINS=http://localhost:3000,http://127.0.0.1:3000
+
+# Admin Credentials (for init/seeding)
+ADMIN_EMAIL="ellarousi@gmail.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)